inst/Description.qml

```diff
@@ -4,7 +4,7 @@ import JASP.Module 	1.0
 Description
 {
 	name		: "jaspCircular"
-	title		: qsTr("Circular Statistics")
+	title		: qsTr("Circular Statistics (update)")
```

Comment:
why this change?

Reply:
Opps, that shouldn't be there. Thanks for spotting it